src/main/resources/view/HelpWindow.css

```diff
@@ -15,7 +19,25 @@
 }
 
 #helpMessageContainer {
-    -fx-background-color: #383838;
+    -fx-background-color: #ffffff;
+}
+
+#moreInfoContainer {
+    -fx-background-color: #ffffff;
+    -fx-font-family: "Arial Rounded MT Bold";
+    -fx-min-height: 80;
+}
+
+.helpMessageWrapper{
+    -fx-background-color: #dedede;
+    -fx-min-height: 5;
+    -fx-min-width: 100;
+    -fx-content-display: CENTER;
+    -fx-alignment: CENTER;
+    -fx-background-radius: 10;
+    -fx-border-radius: 10;
+    -fx-padding: 15;
+
```

Comment:
Nice addition to the UI to fit Client Hub's agreed theme!

Reply:
Good changes to the UI!